content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -116,23 +116,23 @@ Queued experiment '4109ead' for future execution.
 
 ### How are experiments queued?
 
-Queued experiments are created similar to
-[Git stash](https://www.git-scm.com/docs/git-stash). The last experiment queued
-is found in `.git/refs/exps`, and earlier ones are in its [reflog].
+Queued experiments are managed using [dvc-task] and [Celery].
 
-[reflog]:
-  https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefreflogareflog
+[dvc-task]: https://github.com/iterative/dvc-task
+[celery]: https://docs.celeryq.dev/en/stable/index.html
 
 </details>
 
-Run them all with the `--run-all` flag:
+Run them all with `dvc queue start`:
 
 ```dvc
-$ dvc exp run --run-all
+$ dvc queue start
 ...
 ```
 
-> Note that the order of execution is independent of their creation order.
+> Note that in most cases, experiment tasks will be executed in the order that
+> they were added to the queue (First In, First Out), but this order is not
+> guaranteed due to Celery limitations.
```

Comment:
Removing Celery mention from here since it's a hidden implementation detail above.

```suggestion
> Note that in most cases, experiment tasks will be executed in the order that
> they were added to the queue (First In, First Out), but this is not guaranteed.
```